extensions/cornerstone/src/components/WindowLevelActionMenu/Colorbar.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { ReactElement, useCallback, useEffect, useState } from 'react';
+import { SwitchButton } from '@ohif/ui';
+import { StackViewport, VolumeViewport } from '@cornerstonejs/core';
+import { CommandsManager, ServicesManager } from '@ohif/core';
+import { ColorMapPreset } from './WindowLevelActionMenu';
+
+export type ColorbarProps = {
+  viewportId: string;
+  commandsManager: CommandsManager;
+  serviceManager: ServicesManager;
+  displaySets: Array<any>;
+  colorbarProperties: {
+    width: string;
+    colorbarTickPosition: string;
+    colorbarContainerPosition: string;
+    colormaps: Array<ColorMapPreset>;
+    colorbarInitialColormap: string;
+  };
```

Comment:
I see an extremely similar thing in colorbarserivce

```
type ColorbarOptions = {
  position: string;
  colormaps: Array<ColorMapPreset>;
  activeColormapName: string;
  ticks: object;
  width: string;
};
```

are they the same? they should be